<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administración del ITES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tlbUsuarios">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>701</width>
      <height>331</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>CustomContextMenu</enum>
    </property>
    <column>
     <property name="text">
      <string>NombreUsuarios</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contraseñas</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="nombre">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre de Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contrasena">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIngresar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMas">
    <property name="geometry">
     <rect>
      <x>709</x>
      <y>18</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUsuarios"/>
    <addaction name="actionProfesores"/>
    <addaction name="actionAlumnos"/>
    <addaction name="actionMaterias"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <widget class="QMenu" name="menuSobre_su_uso">
     <property name="title">
      <string>Sobre su uso</string>
     </property>
     <addaction name="actionUsuarios_2"/>
    </widget>
    <addaction name="actionAcercade"/>
    <addaction name="menuSobre_su_uso"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionProfesores">
   <property name="text">
    <string>Profesores</string>
   </property>
  </action>
  <action name="actionAlumnos">
   <property name="text">
    <string>Alumnos</string>
   </property>
  </action>
  <action name="actionMaterias">
   <property name="text">
    <string>Materias</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcercade">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionUsuarios_2">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
